nodejs/week1/avg.js

```diff
@@ -0,0 +1,8 @@
+const args = process.argv.slice(2);
+const numbers = args.map(Number);
+const sum = numbers.reduce((acc, num) => acc + num, 0);
+const average = sum / numbers.length;
+console.log(average);
```

Comment:
Hi @Atefeh-Behroozi ,
Overall nice solution,
But something to add are if conditions, which are totally missing, hope you add it
- Check whether there are enough arguments
- Check whether they are numbers